### DISCUSSION TITLE
Smaller breakpoint not overwritten

### DISCUSSION THREAD

**Discussion by tmndrucic-CCA** [tailwindlabs/tailwindcss#6640] [Help] [answered]
Hey,

So I am currently struggling to understand why my tailwind style is not overwritten when changing it for larger screens. I hope someone here can help me with my issue.

So this: `<div className="h-full pb-11 relative xs:pb-24 md:overflow-hidden md:pb-0" key={title}>`

...results to at a 1000px screen:
<img width="470" alt="Bildschirmfoto 2021-12-20 um 12 08 25" src="https://user-images.githubusercontent.com/77392353/146758048-bfa461be-8624-450f-bbcf-16320eb7d382.png">

--> "xs" breakpoint created like this in the config file: 
`extend: {
      screens: {
        xxs: '320px',
        xs: '360px',
      },`

Thank you in advance! :)

**Comment by tmndrucic-CCA** [comment]
So I also stumbled over https://github.com/tailwindlabs/tailwindcss/discussions/6368 and it seems to me also that only padding classes are not overwritten.. is there a solution for this?

**Comment by adamwathan** [comment, marked as answer]
Your `xxs` and `xs` breakpoints are being defined at the end of your screens list, check the docs here:

https://tailwindcss.com/docs/screens#adding-smaller-breakpoints

**Reply by tmndrucic-CCA** [reply]
Thanks!!
